cosmic/microservices/cosmic-bill-viewer/values.yaml

```diff
@@ -0,0 +1,20 @@
+# Default values for cosmic-bill-viewer.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  name: missioncriticalcloud/cosmic-bill-viewer
+  tag: latest
+  pullPolicy: IfNotPresent
```

Comment:
That's only the default value. We can always overwrite it with an environment `values.yaml` file. In my opinion it makes sense to have `IfNotPresent` as default because we are supposed to be using tags anyway (`1.0.0`, `1.1.0`, etc).